Tasks/Npm/npminstall.ts

```diff
@@ -1,9 +1,13 @@
 import * as path from 'path';
-
 import * as tl from 'vsts-task-lib/task';
-
 import * as npmCustom from './npmcustom';
+import { NpmTaskInput} from './constants';
 
 export async function run(): Promise<void> {
-    return npmCustom.run('install');
+    let Command = 'install';
+    let verboseCheck = tl.getBoolInput(NpmTaskInput.Verbose, false);
+    if (verboseCheck) {
+        Command += " --verbose";
```

Comment:
Since this is changing the way npm runs, I'd rather have this as part of the `npmtoolrunner`, it would also be good to set this as an environment level variable, rather than messing with the command itself.